lib/node.js

```diff
@@ -182,7 +182,7 @@ class Node {
 
     switch (data.event) {
       case 'autorefresh':
-        this.redis
+        return this.redis
```

Comment:
Inconsistent function return value: you cannot safely use `sync.then()` because it might not return a thenable.

You could add a `return Bluebird.resolve();` line after the switch-case block to fix that.

Reply:
:+1: